proplet/docs/TEE-WORKLOAD-TESTING.md

```
@@ -0,0 +1,1004 @@
+cd propeller/proplet
+
+# Build with CDH support for encrypted workloads
+cargo build --release --features cdh
+
+# Install
+sudo cp target/release/proplet /usr/local/bin/
+```
+
+### Port architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)